media-sound/flac123/flac123-0.0.12.recipe

```diff
@@ -0,0 +1,51 @@
+SUMMARY="A command-line program for playing FLAC audio files"
+DESCRIPTION="flac-tools aims to be a complete set of tools for playing and \
+manipulating audio files stored in the FLAC format.  flac123 is the only \
+component of flac-tools at this time."
+HOMEPAGE="http://flac-tools.sourceforge.net/"
+COPYRIGHT="2003-2017 Jake Angerman"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="http://downloads.sourceforge.net/flac-tools/flac123-$portVersion-release.tar.gz"
```

Comment:
You can change this to
```
SOURCE_URI="https://downloads.sourceforge.net/flac-tools/flac123-$portVersion-release.tar.gz" 
```
or
```
SOURCE_URI="https://downloads.sf.net/flac-tools/flac123-$portVersion-release.tar.gz" 
```
since SourceForge is accepting https traffic and, for flac123, it is not even redirecting it to http ;-)
(Well, maybe I was lucky and got an https-enabled mirror ;-)
Anyway, https will work for everyone. And if SF redirects to http that is not a problem. You can use https for the SOURCE_URI.